synapse/lib/storm.py

```diff
@@ -3199,7 +3199,26 @@ async def execStormCmd(self, runt, genr):
 
                 async with snap.getEditor() as editor:
 
-                    proto = await editor.addNode(node.ndef[0], node.ndef[1], props=node.props)
+                    proto = await editor.addNode(node.ndef[0], node.ndef[1])
+
+                    for name, valu in node.props.items():
+
+                        prop = node.form.prop(name)
+                        if prop.info.get('ro'):
+                            if name == '.created':
+                                proto.props['.created'] = valu
```

Comment:
whats the difference between setting `proto.props[.created]` vs `proto.set(name, valu)` ?